js/content/store.js

```diff
@@ -171,77 +171,85 @@ class StorePageClass {
 
         let text = "";
         for (let node of document.querySelectorAll(".game_area_sys_req, #game_area_legal, .game_details, .DRM_notice")) {
-            text += node.innerHTML;
+            text += node.textContent.toLowerCase();
         }
-        let uppercased = text.toUpperCase();
 
         // Games for Windows Live detection
         let gfwl =
-               uppercased.includes("GAMES FOR WINDOWS LIVE")
-            || uppercased.includes("GAMES FOR WINDOWS - LIVE")
-            || text.includes("Online play requires log-in to Games For Windows")
-            || text.includes("INSTALLATION OF THE GAMES FOR WINDOWS LIVE SOFTWARE")
-            || text.includes("Multiplayer play and other LIVE features included at no charge")
+               text.includes("games for windows live")
+            || text.includes("games for windows - live")
+            || text.includes("online play requires log-in to games for windows")
+            || text.includes("installation of the games for windows live software")
+            || text.includes("multiplayer play and other live features included at no charge")
             || text.includes("www.gamesforwindows.com/live");
 
         // Ubisoft Uplay detection
         let uplay =
-               text.includes("Uplay")
-            || text.includes("Ubisoft Account");
+               text.includes("uplay")
+            || text.includes("ubisoft account");
 
         // Securom detection
-        let securom = text.includes("SecuROM");
+        let securom = text.includes("securom");
 
         // Tages detection
         let tages =
-                text.match(/\b(tages|solidshield)\b/i)
-            && !text.match(/angebote des tages/i);
+                text.match(/\b(tages|solidshield)\b/)
+            && !text.match(/angebote des tages/);
 
         // Stardock account detection
-        let stardock = text.includes("Stardock account");
+        let stardock = text.includes("stardock account");
 
         // Rockstar social club detection
         let rockstar =
-               text.includes("Rockstar Social Club")
-            || text.includes("Rockstar Games Social Club");
+               text.includes("rockstar social club")
+            || text.includes("rockstar games social club");
 
         // Kalypso Launcher detection
-        let kalypso = text.includes("Requires a Kalypso account");
+        let kalypso = text.includes("requires a kalypso account");
 
         // Denuvo Antitamper detection
-        let denuvo = text.match(/\bdenuvo\b/i);
-
-        // Detect other DRM
-        let drm =
-                text.includes("3rd-party DRM")
-            && !text.match(/No (3rd|third)[- ]party DRM/i);
```

Comment:
I removed this because if it was used to match contents in the DRM notice, the text is translated by Steam, so it'll likely be `false` for all languages other than English.